README.md

```diff
@@ -260,6 +259,18 @@ Out of the box, the health check package provides:
  * BasicAuth - Requires that basic auth credentials be sent in order to see full status
  * AddHeaders - Adds X-check-status headers to the response, so you can avoid having to parse JSON
 
+### Checks
+
+##### Scheduler Health Check
+
+The scheduler health check works by using a time limited cache key on your project every minute. You will need to register the
```

Comment:
Yeah, I'd seen it was possible and there were various opinions. Quite a few answers though weren't keen on packages automatically scheduling as they liked to keep track of that themselves.

Spatie make you schedule yourself rather than have config options (https://spatie.be/docs/laravel-backup/v6/installation-and-setup#scheduling) so I'm leaning that way

Reply:
Yeah, that's true. I wouldn't like stuff being scheduled that I didn't know about. Let's do it manually then.